src/components/Explore/TokenRow.tsx

```diff
@@ -462,35 +462,32 @@ export default function LoadedRow({
   const tokenSymbol = token?.symbol
   const tokenData = data[tokenAddress]
   const [favoriteTokens, updateFavoriteTokens] = useAtom(favoritesAtom)
+  const theme = useTheme()
   // TODO: remove magic number colors
   const tokenPercentChangeInfo = (
     <>
       {tokenData.delta}%
       <ArrowCell>
         {Math.sign(tokenData.delta) > 0 ? (
-          <ArrowUpRight size={16} color={'#57bd0f'} />
+          <ArrowUpRight size={16} color={theme.green1} />
         ) : (
-          <ArrowDownRight size={16} color={'red'} />
+          <ArrowDownRight size={16} color={theme.red1} />
         )}
       </ArrowCell>
     </>
   )
-  const theme = useTheme()
 
   /* handle favorite token logic */
-  const isTokenFavorited = atom<boolean>(favoriteTokens.includes(tokenAddress))
-  const [tokenFavorited, setTokenFavorite] = useAtom(isTokenFavorited)
   const toggleFavoriteToken = () => {
-    let updatedFavoriteTokens = favoriteTokens
-    if (tokenFavorited) {
+    let updatedFavoriteTokens = favoriteTokens.slice()
```

Comment:
`[...favoritedTokens]`

Reply:
<img width="628" alt="Screen Shot 2022-07-05 at 2 19 19 PM" src="https://user-images.githubusercontent.com/62825936/177390918-a11050b4-8a5d-4571-bd20-0ef533148763.png">

Reply:
for some reason this is the same bug where it doesn't re-render